### ISSUE TITLE
Latest version of the plugin does not work in Maven 3.8.2

### ISSUE THREAD

**Issue by stephaniewang526** [mojohaus/flatten-maven-plugin#230]
We are getting this error across googleapis repos:
```
Failed to execute goal org.codehaus.mojo:flatten-maven-plugin:1.2.7:flatten (flatten) on project google-cloud-spanner: failed to create a clean pom: unable to create flattened dependencies: caught exception when flattening dependencies: Failed to read artifact descriptor for org.apache.httpcomponents:httpclient::4.5.13: Could not find artifact org.apache:apache:pom:21
```
Example full trace: https://github.com/googleapis/java-bigquery/pull/1556/checks?check_run_id=3498903147 

The latest version of the plugin seems to not work in the latest version of Maven.

**Comment by jorsol** [comment]
Looks like an issue in Maven: https://lists.apache.org/thread.html/rda29028b2c8985f3b94e721d3014a948b312fbddf95ffaa4971acc03%40%3Cusers.maven.apache.org%3E

Is potentially being fixed in Maven 3.8.3

**Comment by matthewayne** [comment]
I had the same issue on with the Java library for GCP's Document AI. I just verified that the problem is fixed on the Document AI library with Maven 3.8.3 (just release ~14 hours ago)
